packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -249,8 +243,6 @@ function* getCoinflowPurchaseMetadata({
     quantity: 1,
     rawProductData: {
       priceUSD: price / 100,
-      extraAmountUSD: extraAmount ? extraAmount / 100 : 0,
-      usdcRecipientSplits: splits,
```

Comment:
Flagging that I'm changing what we send to coinflow as part of raw product data. This allows us to focus on the product rather than the price in these, and cleans up the respective sagas greatly - but I don't know if this will trigger Coinflow problems

Cc: @raymondjacobson / @raymondjacobson

Reply:
I think it's ok to remove this, but we should really try to keep price in here

Reply:
keeping the track/album price in here, but note that it won't include "extraAmount" - that ok?